src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocContentLocationCheck.java

```diff
@@ -0,0 +1,240 @@
+ * </p>
+ * <ul>
+ * <li>
+ * Property {@code location} - Specify the policy on placement of the Javadoc content.
+ * Default value is {@code second_line}.
+ * </li>
+ * </ul>
+ * <p>
+ * By default Check validate that the Javadoc content starts from the second line:
+ * </p>
+ * <pre>
+ * &lt;module name="JavadocContentLocationCheck"/&gt;
+ * </pre>
+ * <p>
+ * This setting produces a violation for each multi-line comment starting
+ * on the same line as the initial asterisks:
+ * </p>
+ * <pre>
+ * &#47;** This comment causes a violation because it starts from the first line
+ *   * and spans several lines.
+ *   *&#47;
+ * &#47;**
+ *   * This comment is OK because it starts from the second line.
+ *   *&#47;
+ * &#47;** This comment is OK because it is on the single line. *&#47;
+ *</pre>
+ * <p>
+ * To ensure that Javadoc content starts from the first line:
+ * </p>
+ * <pre>
+ * &lt;module name="JavadocContentLocationCheck"&gt;
+ *   &lt;property name="location" value="FIRST_LINE"/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * This setting produces a violation for each comment not
+ * starting on the same line as the initial asterisks:
+ * </p>
+ * <pre>
+ * &#47;** This comment is OK because it starts on the first line.
+ *    * There may be additional text.
+ *    *&#47;
+ * &#47;**
+ *   * This comment causes a violation because it starts on the second line.
+ *   *&#47;
+ * &#47;** This single-line comment also is OK. *&#47;
+ * </pre>
+ *
+ * @since 8.27
+ */
+@StatelessCheck
+public class JavadocContentLocationCheck extends AbstractCheck {
+
+    /**
+     * A key is pointing to the warning message text in "messages.properties" file.
+     */
+    public static final String MSG_JAVADOC_CONTENT_FIRST_LINE = "javadoc.content.first.line";
+
+    /**
+     * A key is pointing to the warning message text in "messages.properties" file.
+     */
+    public static final String MSG_JAVADOC_CONTENT_SECOND_LINE = "javadoc.content.second.line";
+
+    /**
+     * Specify the policy on placement of the Javadoc content.
+     */
+    private JavadocContentLocation location = JavadocContentLocation.SECOND_LINE;
+
+    @Override
+    public int[] getRequiredTokens() {
+        return new int[] {
+            TokenTypes.BLOCK_COMMENT_BEGIN,
+        };
+    }
+
+    @Override
+    public int[] getAcceptableTokens() {
+        return getRequiredTokens();
+    }
+
+    @Override
+    public int[] getDefaultTokens() {
+        return getRequiredTokens();
+    }
+
+    @Override
+    public boolean isCommentNodesRequired() {
+        return true;
+    }
+
+    @Override
+    public void visitToken(DetailAST ast) {
+        if (isMultilineComment(ast) && JavadocUtil.isJavadocComment(ast)) {
+            final String commentContent = JavadocUtil.getJavadocCommentContent(ast);
+            final int blankLines = countBlankLines(commentContent);
+            if (blankLines >= 0) {
```

Comment:
Please explain reason to be aware that we move no blank lines (-1) ?
It is not possible as you have `(isMultilineComment(ast)`

Reply:
`-1` is for the case then the comment contains only blank lines.
It shouldn't happen in real life. I can remove it if you think it's unnecessary.

Reply:
upd. Removed workaround for blank-only comments. They shouldn't exist, so there's no reason to handle them.

Reply:
completely empty comments is result of auto-generation of IDE, I think there are bunch of them in code.
If javadoc comment is empty .... we do not know what location will be used to put a content.
So it is  better to be silent about such cases.

Reply:
done. Reverted back to 3c79ac4a6c00d14d20dd8b249aec33f466056fad